src/include/kdbhelper.h

```diff
@@ -31,7 +31,7 @@ void elektraFree (void * ptr);
 int elektraRealloc (void ** buffer, size_t size);
 
 char * elektraStrDup (const char * s);
-char * elektraStrNDup (const char * s, size_t l);
+char * elektraMemDup (const char * s, size_t l);
```

Comment:
@kodebach wouldn't it make sense, in the spirit of `mem*()` functions to switch to `const void * s` as well? I know it is not mentioned in the original issue, but still, it would be easy right here.

```suggestion
char * elektraMemDup (const void * s, size_t l);
```

Reply:
Yes, but then we should also return a `void *` instead of a `char *`.

Reply:
True. Let's fix that including the docu in a later PR.